glass/src/libnt/native/include/glass/networktables/NTAlerts.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <string>
+#include <string_view>
+#include <vector>
+
+#include <networktables/NetworkTableInstance.h>
+#include <networktables/StringArrayTopic.h>
+
+#include "glass/other/Alerts.h"
+
+namespace glass {
+
+class NTAlertsModel : public AlertsModel {
+ public:
+  static constexpr const char* kType = "Alerts";
+
+  // path is to the table containing ".type", excluding the trailing /
+  explicit NTAlertsModel(std::string_view path);
+  NTAlertsModel(nt::NetworkTableInstance inst, std::string_view path);
+
+  const std::vector<std::string>& GetInfos() override { return m_infosValue; }
+
+  const std::vector<std::string>& GetWarnings() override {
+    return m_warningsValue;
+  }
+
+  const std::vector<std::string>& GetErrors() override { return m_errorsValue; }
+
+  void Update() override;
+  bool Exists() override;
+  bool IsReadOnly() override { return false; }
```

Comment:
I'm not familiar with Glass- Could someone explain if this is accurate? I don't see any code to respond to changes on the client side.